src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -162,7 +190,7 @@ export default class MutationAssessorColumnFormatter
         const msaLink = MutationAssessorColumnFormatter.maLink(maData.msa);
         const pdbLink = MutationAssessorColumnFormatter.maLink(maData.pdb);
 
-        if (maData.score)
+        if (maData.score !== undefined )
```

Comment:
I wanted to make sure that scores of 0 would still display, but will add in a case to check for null!